specification/app/resource-manager/Microsoft.App/stable/2025-01-01/SessionPools.json

```diff
@@ -0,0 +1,771 @@
+              "$ref": "#/definitions/CustomContainerTemplate"
+            },
+            "sessionNetworkConfiguration": {
+              "description": "The network configuration of the sessions in the session pool.",
+              "$ref": "#/definitions/SessionNetworkConfiguration"
+            },
+            "poolManagementEndpoint": {
+              "description": "The endpoint to manage the pool.",
+              "type": "string",
+              "format": "uri",
+              "readOnly": true
+            },
+            "provisioningState": {
+              "description": "Provisioning state of the session pool.",
+              "enum": [
+                "InProgress",
+                "Succeeded",
+                "Failed",
+                "Canceled",
+                "Deleting"
+              ],
+              "type": "string",
+              "readOnly": true,
+              "x-ms-enum": {
+                "name": "SessionPoolProvisioningState",
+                "modelAsString": true
+              }
+            },
+            "managedIdentitySettings": {
+              "description": "Optional settings for a Managed Identity that is assigned to the Session pool.",
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/ManagedIdentitySetting"
+              },
+              "x-ms-identifiers": [
+                "identity"
+              ]
+            }
+          },
+          "x-ms-client-flatten": true
+        }
+      }
+    },
+    "ManagedIdentitySetting": {
+      "description": "Optional settings for a Managed Identity that is assigned to the Session pool.",
+      "type": "object",
+      "required": [
+        "identity"
+      ],
+      "properties": {
+        "identity": {
+          "description": "The resource ID of a user-assigned managed identity that is assigned to the Session Pool, or 'system' for system-assigned identity.",
+          "type": "string"
+        },
+        "lifecycle": {
+          "description": "Use to select the lifecycle stages of a Session Pool during which the Managed Identity should be available.",
+          "enum": [
+            "None",
+            "Main"
+          ],
+          "type": "string",
+          "default": "None",
+          "x-ms-enum": {
+            "name": "IdentitySettingsLifeCycle",
+            "modelAsString": true
+          }
+        }
+      }
+    },
+    "SessionPoolUpdatableProperties": {
+      "description": "Container App session pool updatable properties.",
+      "type": "object",
+      "properties": {
+        "identity": {
+          "description": "Managed identities needed by a session pool to interact with other Azure services to not maintain any secrets or credentials in code.",
+          "$ref": "../../../../../common-types/resource-management/v5/managedidentity.json#/definitions/ManagedServiceIdentity"
+        },
```

Comment:
[ARMBlockingComment] You must support PATCH for tags for a tracked resource

Reply:
Hi @raosuhas PATCH for tags is added, could you please take a look again, thanks.